src/app/services/items.service.ts

```diff
@@ -0,0 +1,52 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { BehaviorSubject } from 'rxjs';
+import { IItem } from '../app.models';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class ItemsService {
```

Comment:
You shouldn't rename this service to ItemsService. It should be named WatchesService as it used to.